files/zh-cn/web/api/window/devicemotion_event/index.md

```diff
@@ -1,23 +1,56 @@
 ---
-title: Window.ondevicemotion
+title: Window：devicemotion 事件
 slug: Web/API/Window/devicemotion_event
+l10n:
+  sourceCommit: 6d4f585b94068dc0dfd733047fb2229dca38b1eb
 ---
 
-{{ ApiRef() }}
+{{APIRef("Device Orientation Events")}}{{securecontext_header}}
 
-## 摘要
+**`devicemotion`** 事件每隔一定时间触发一次，显示设备当时受到的物理加速度力的大小。如果有的话，它还会提供有关旋转速率的信息。
```

Comment:
I'm not sure about the English content: why there is a word "force". Could you check this?

```suggestion
**`devicemotion`** 事件每隔一定时间触发一次，显示设备当时受到的物理加速度的大小。如果有的话，它还会提供有关旋转速率的信息。
```

Reply:
> I'm not sure about the English content: why there is a word "force". Could you check this?

I found it somehow confusing too 🤔

Reply:
mdn/content#32153